install/terraform/open-match/secure-gke.tf

```diff
@@ -59,6 +59,22 @@ variable "gcp_machine_type" {
   default = "n1-standard-4"
 }
 
+# Enable Kubernetes API
+resource "google_project_service" "kubernetes_api" {
+  project     = "${var.gcp_project_id}"
+  service     = "container.googleapis.com"
+  
+  disable_dependent_services= true
+}
+
+# Enable Cloud Resource Manager API
+resource "google_project_service" "cloud_resource_manager_api" {
+  project     = "${var.gcp_project_id}"
+  service     = "cloudresourcemanager.googleapis.com"
+  
+  disable_dependent_services= true
```

Comment:
`disable_dependent_services` looks very risky. I'm not sure we should enable this because it can lead to data loss.